test/partition.test.js

```diff
@@ -0,0 +1,168 @@
+// Copyright IBM Corp. 2016,2019. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+require('./init.js');
+const _ = require('lodash');
+const should = require('should');
+const DEFAULT_MODEL_VIEW = 'loopback__model__name';
+let Product, db, connector;
+
+const config = {
+  url: process.env.CLOUDANT_URL,
+  username: process.env.CLOUDANT_USERNAME,
+  password: process.env.CLOUDANT_PASSWORD,
+  database: process.env.CLOUDANT_PARTITIONED_DATABASE,
+  plugin: 'retry',
+  retryAttempts: 10,
+  retryTimeout: 50,
+};
+
+describe('cloudant - partitioned db', () => {
+  before(function(done) {
+    db = global.getDataSource(config);
+    connector = db.connector;
+    db.automigrate(done);
+  });
+
+  it('property level - create global index by default', (done) => {
+    Product = db.define('Product', {
+      prodName: {type: String, index: true},
+      desc: {type: String},
+    });
+    db.autoupdate('Product', (err) => {
+      if (err) return done(err);
+      connector.getIndexes(connector.getDbName(connector), (e, results) => {
+        if (e) return done(e);
+        const indexes = results.indexes;
+        const indexName = 'prodName_index';
+        should.exist(indexes);
+
+        const index = _.find(indexes, function(index) {
+          return index.name === indexName;
+        });
+        should.exist(index);
+        index.name.should.equal(indexName);
+        index.def.fields[0]['prodName'].should.equal('asc');
+        index.def.fields[1][DEFAULT_MODEL_VIEW].should.equal('asc');
+        // should be a global index
+        index.partitioned.should.equal(false);
+        done();
+      });
+    });
+  });
+
+  it('index entry - create global index by default', (done) => {
+    Product = db.define('Product', {
+      prodName: {type: String},
+      desc: {type: String},
+    }, {
+      indexes: {
```

Comment:
not really about this test case.. Do we have any tests to show after we've created a database using the `partitioned:true` option, the `indexes` look like this?

Reply:
Discussed it with @jannyHou. This test case is about if someone specify an `indexes` for the product, it should be preserved after putting it into the database. 
For the `partitioned:true` "property", it is at the global level, so if we don't specify any `indexes`, there will be no difference for the indexes between a partitioned database and a non-partitioned database.